src/components/__tests__/app.test.js

```diff
@@ -0,0 +1,28 @@
+// React Libraries
+import React from 'react';
+
+// third-party libraries
+import { shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+// modules importation
+import Home from '../Home';
+
+configure({ adapter: new Adapter() });
+
+
```

Comment:
NIce work @koiic , you should remove this blank line to make the code more readable

Reply:
Thanks...I will work on it